static/app/components/u2f/u2finterface.tsx

```diff
@@ -166,9 +190,18 @@ class U2fInterface extends React.Component<Props, State> {
         this.submitU2fResponse(promise);
       }
     } else if (this.props.flowMode === 'enroll') {
-      const {registerRequests, registeredKeys} = this.props.challengeData;
-      promise = u2f.register(registerRequests as any, registeredKeys as any);
-      this.submitU2fResponse(promise);
+      const {organization} = this.props;
+      if (organization.features.includes('webauthn-register')) {
+        const challengeArray = base64urlToBuffer(
+          this.props.challengeData.webAuthnRegisterData
+        );
+        const challenge = cbor.decodeAllSync(challengeArray);
+        this.webAuthnRegister(challenge[0].publicKey);
```

Comment:
would add another note here as to why you only register the first one in the array